src/frontend/mame/luaengine.cpp

```diff
@@ -23,6 +23,7 @@
 #include "natkeyboard.h"
 #include "softlist.h"
 #include "uiinput.h"
+#include "imagedev/cassette.h"
```

Comment:
Please keep includes from `src/devices` visually separate from the ones from `src/emu` (they should come before the `src/emu` ones, too).